azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -450,8 +440,10 @@ private void setupKeyStore() throws ExecutorManagerException {
         logger.error("Failed to Prefetch KeyStore");
         throw new ExecutorManagerException("Failed to Prefetch KeyStore");
       }
+      logger.info("In-memory Keystore is setup, delete the cert file");
```

Comment:
Shouldn't this be behind config to support in memory keystore or now? Delete should also not happen if it is not in memory.

Reply:
The entire code to setup KeyStore is under config.

Reply:
How will hadoopSecurityManager created if this method will not be called?

Reply:
HadoopSecurityManager is created per job with the introduction of Robin. Here it is created specifically for fetching the KeyStore. It is never used again. It is not singleton anymore.